cli/admin/src/test/java/io/pravega/cli/admin/debug/ReaderGroupStreamFileParsingCommandTest.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.cli.admin.debug;
+
+import io.pravega.cli.admin.AdminCommandState;
+import io.pravega.cli.admin.utils.TestUtils;
+import lombok.Cleanup;
+import lombok.val;
+import org.apache.commons.io.FileUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class ReaderGroupStreamFileParsingCommandTest {
+
+    @Test
+    public void testPasswordFileCreatorCommand() throws Exception {
+        final String inputFileName = "input";
+        final String outputFileName = "output";
+        byte[] bytes = {0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
Is there a way to test with a more realistic content that actually has some info about a reader group?